PowerToys.sln

```diff
@@ -96,192 +96,90 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ImageResizer.Test", "src\mo
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU

```

Comment:
Were this changes generated by VS automatically?

Reply:
I think they were generated by VS automatically and they got added in my previous PR accidentally. I saw that the current master doesn't have the x86 options so I removed it in this PR to match that.